code/modules/chemistry/Chemistry-Machinery.dm

```diff
@@ -304,7 +304,6 @@ TYPEINFO(/obj/machinery/chem_shaker)
 #else
 	icon_state = "orbital_shaker"
 #endif
-	density = 1
```

Comment:
?

Reply:
glass recycler is undense, it's also a tabletop device so i reasoned that it shouldn't be dense either